ProteoformSuiteGUI/ProteoformFamilies.cs

```diff
@@ -266,5 +267,46 @@ private void cb_buildAsQuantitative_CheckedChanged(object sender, EventArgs e)
             cb_boldLabel.Enabled = cb_buildAsQuantitative.Checked;
             cb_moreOpacity.Enabled = false; //not fully implemented
         }
+
+        private void btn_inclusion_list_all_families_Click(object sender, EventArgs e)
```

Comment:
Sure, list of m/z and charge states for if you want to run a targeted TD run. The MS lets you import a .txt file in this format. MRS mentioned you guys may want to validate some yeast stuff with one?